tests/infrastructure/downward-metrics.go

```diff
@@ -69,7 +59,7 @@ var _ = DescribeInfra("downwardMetrics", func() {
 		Expect(libinfra.GetHostnameFromMetrics(metrics)).To(Equal(vmi.Status.NodeName))
 
 	},
-		Entry("[test_id:6535]using a disk", libvmi.WithDownwardMetricsVolume("vhostmd"), libinfra.GetDownwardMetricsDisk),
+		Entry("[test_id:6535]using a disk", decorators.Conformance, libvmi.WithDownwardMetricsVolume("vhostmd"), libinfra.GetDownwardMetricsDisk),
```

Comment:
Would you explain in the PR text why you consider this test as covering a "core functionality"? 
I find downward metrics as an interesting feature, which is useful in several use case, but not part of the core of kuebvirt

Reply:
Well, actually there is a bigger problem that I missed: downward metrics are still behind a FG, which makes it directly ineligible to be part of the conformance :) Thank you!
/close